heclib/heclib_f/Makefile

```diff
@@ -4,16 +4,26 @@ SRC_DIRS := ./src
 
 
 UNAME_S := $(shell uname -s)
-FOR_VER := $(shell gfortran -dumpversion)
+UNAME_M := $(shell uname -m)
+FOR_VER := $(shell gfortran -dumpversion|cut -c1,2)
 
 ifeq ($(UNAME_S),Linux)
 	FWARNINGS=-Warray-bounds -Wc-binding-type -Wcharacter-truncation
+	ifeq "$(FOR_VER)" "10"
```

Comment:
I'm going going nuts of is this saying "if gfortran 10,11,or 12 " then use -fallow-argument-mismatch.

Could be simplier logic, also the indentation is way off.

Reply:
Also gnu make only supports conditionals `ifeq `and `ineq`, so we can't test for all the version in range in a single test. :(
The apparent indentation was good in vscode on my macbook, but I'll fix it.

Reply:
Created a `F_GE_10` variable via a shell test to simplify the fortran version stuff.